bpf/lib/environ_conf.h

```diff
@@ -25,6 +25,8 @@ struct tetragon_conf {
 	__u32 tg_cgrp_level; /* Tetragon cgroup level */
 	__u64 tg_cgrpid; /* Tetragon current cgroup ID to avoid filtering blocking itself */
 	__u64 cgrp_fs_magic; /* Cgroupv1 or Cgroupv2 */
+	__u8 use_perf_ring_buf; /* Use perf ring buffer rather than the bpf ring buffer */
```

Comment:
idea: you could use global const in place and then profit from dead code elimination (from 5.4).